rust-wasm-id-allocator/id-types/src/session_id.rs

```diff
@@ -40,13 +40,27 @@ impl SessionId {
             }
         }
     }
+}
+
+/// Creates a new SessionId from the supplied UUID in number form.
+/// Free function to keep it internal.
```

Comment:
What does this mean? (I'm assuming this is the motivation for this change)

Reply:
Moving it out of the impl block (thus making it a "free" function) means that the function isn't usable by consumers of the core/wasm crates since they only re-export the ID type.